conda-recipe/recipe.yaml

```diff
@@ -77,7 +77,7 @@ outputs:
         - wrapt >=1.13.3
       run_constraints:
         - pyside2 >=5.15.1
-        - pyside6 <6.5|>=6.7
+        - pyside6 >=6.7
```

Comment:
@psobolewskiPhD given [your comment](https://github.com/napari/packaging/issues/269#issuecomment-3120654140) in #269, should this not be `>=6.7.1`? (ditto for our pyproject.toml, really...)

Reply:
honestly i'm not sure.

Reply:
I think it couldn't hurt: #274